setup/setup.sh

```diff
@@ -0,0 +1,217 @@
+    
+    # Set automatic update check to true, but disable automatic installation
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist AutomaticCheckEnabled -bool true
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist AutomaticDownload -bool true
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist AutomaticallyInstallMacOSUpdates -bool false
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist CriticalUpdateInstall -bool false
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist ConfigDataInstall -bool false
+    
+    # Disable automatic installation of app updates from App Store
+    sudo defaults write /Library/Preferences/com.apple.commerce.plist AutoUpdate -bool false
+    sudo defaults write /Library/Preferences/com.apple.commerce.plist AutoUpdateRestartRequired -bool false
+    
+    log "macOS updates configured for Download Only mode"
+}
+
+# Download and install Orka VM Tools
+install_orka_vm_tools() {
+    log "Installing Orka VM Tools version ${ORKA_VM_TOOLS_VERSION}..."
+    
+    local pkg_url="https://orka-tools.s3.amazonaws.com/orka-vm-tools/official/${ORKA_VM_TOOLS_VERSION}/orka-vm-tools.pkg"
+    local pkg_path="/tmp/orka-vm-tools.pkg"
+    local pkg_name="orka-vm-tools.pkg"
+    
+    # Download the Orka VM Tools package
+    log "Downloading Orka VM Tools from $pkg_url..."
+    curl -fsSL "$pkg_url" -o "$pkg_path" || error "Failed to download Orka VM Tools package"
+    
+    # Verify the download was successful
+    if [[ ! -f "$pkg_path" ]]; then
+        error "Orka VM Tools package not found after download"
+    fi
+    
+    # Install the package
+    log "Installing Orka VM Tools package..."
+    sudo installer -pkg "$pkg_path" -target / || error "Failed to install Orka VM Tools package"
+    
+    # Clean up installer
+    rm -f "$pkg_path"
+    
+    log "Orka VM Tools ${ORKA_VM_TOOLS_VERSION} installed successfully"
+}
+
+# Download and run the setup-sys-daemon.sh script
+setup_sys_daemon() {
+    log "Setting up system daemon..."
+    
+    local script_url="https://raw.githubusercontent.com/macstadium/packer-plugin-macstadium-orka/main/guest-scripts/setup-sys-daemon.sh"
+    
+    # Download and execute the script directly
+    curl -fsSL "$script_url" | sudo bash || error "Failed to run setup-sys-daemon.sh"
+    
+    log "System daemon setup completed"
+}
+
+# Post-install system cleanup
+cleanup_system() {
+    log "Cleaning up system..."
+    
+    # Close all applications (except essential system apps)
+    osascript -e 'tell application "System Events" to set quitapps to name of every application process whose background only is false'
+    osascript -e 'repeat with apps in quitapps
+        if apps is not in {"Finder", "loginwindow"} then
+            tell application apps to quit
+        end if
+    end repeat' 2>/dev/null || true
```

Comment:
This does not work for me:

<img width="1288" height="893" alt="Screenshot 2025-10-03 at 12 25 38 PM" src="https://github.com/user-attachments/assets/b6942e5c-b1df-4a44-819e-4bd02c490ecc" />

After the script completes and system reboots, Terminal, Safari and System Preferences are all still opened